### PR TITLE
WSL R and RStudio instructions

### LINKED ISSUE #253: Update R/RStudio windows instructions for WSL
We need to update R/RStudio install instructions for windows users. 
After they install WSL, they will need to install all the R things _on the Ubuntu side_:

- install linux R
- install RStudio _Server_ (since they can't use Desktop)
  -  also explain how to launch server

Somewhere towards the top, remind them that we are recommending this because OpenScPCA's supported operating systems are mac and linux, not windows, so this is how they can use windows in a way where we can help them the most. They can still their regular Windows setup but may not be able to do all the things in openscpca.

### PR DESCRIPTION
**Pull request by sjspielman**
Closes #253 

This PR updates the R/RStudio instructions for WSL. Probably needs some phrasing tweaks throughout, but the process is about as streamlined as I was able to achieve now!

### CODE REVIEW

File: docs/technical-setup/environment-setup/install-r-rstudio.md [hunk 1]
```diff
@@ -42,26 +42,106 @@ These tools are available from the [R for macOS website](https://mac.r-project.o
...
+1. Copy and paste this code into the Ubuntu terminal to install R:
+
+    ```sh
+    # install a few helper packages needed to install R and certain R packages
+    sudo apt install --no-install-recommends software-properties-common dirmngr libssl-dev
+
+    # add the signing key (by Michael Rutter) for these repos
+    wget -qO- https://cloud.r-project.org/bin/linux/ubuntu/marutter_pubkey.asc | sudo tee -a /etc/apt/trusted.gpg.d/cran_ubuntu_key.asc
+
+    # add the R 4.0 repo from CRAN -- adjust 'focal' to 'groovy' or 'bionic' as needed
```
Comment (jashapiro) [inline docs/technical-setup/environment-setup/install-r-rstudio.md:58]: I would not expect anyone to know what this means.
```suggestion
    # add the R 4.0 repo from CRAN 
```

you could als use `sudo add-apt-repository "deb https://cloud.r-project.org/bin/linux/ubuntu $(lsb_release -cs)-cran40/"` to automatically set the version.

File: docs/technical-setup/environment-setup/install-r-rstudio.md [hunk 2]
```diff
@@ -42,26 +42,106 @@ These tools are available from the [R for macOS website](https://mac.r-project.o
...
+#### Using the RStudio Server
+
+To access the RStudio Server, the server must be running.
+You can start and stop the server by running the following lines in Ubuntu terminal.
+We recommend that you stop the server after each coding session:
+
+- Start RStudio Server
+
+    ```sh
+    sudo systemctl start rstudio-server.service
```
Comment (jashapiro) [inline docs/technical-setup/environment-setup/install-r-rstudio.md:127]: I think this simpler version should work?
```suggestion
    sudo rstudio-server start
```

File: docs/technical-setup/environment-setup/install-r-rstudio.md [hunk 3]
```diff
@@ -42,26 +42,106 @@ These tools are available from the [R for macOS website](https://mac.r-project.o
...
+    ```
+
+- Stop RStudio Server
+
+    ```sh
+    sudo systemctl stop rstudio-server.service
```
Comment (jashapiro) [inline docs/technical-setup/environment-setup/install-r-rstudio.md:133]: See above
```suggestion
    sudo rstudio-server stop
```

File: docs/technical-setup/environment-setup/install-r-rstudio.md [hunk 4]
```diff
@@ -42,26 +42,106 @@ These tools are available from the [R for macOS website](https://mac.r-project.o
...
+    ```sh
+    rm rstudio-server-latest-amd64.deb
+    ```
+
+
+#### Using the RStudio Server
+
+To access the RStudio Server, the server must be running.
+You can start and stop the server by running the following lines in Ubuntu terminal.
+We recommend that you stop the server after each coding session:
```
Comment (jashapiro) [inline docs/technical-setup/environment-setup/install-r-rstudio.md:122]: Do we? I'm not sure there is much cost to leaving it open.

File: docs/technical-setup/environment-setup/install-r-rstudio.md [hunk 5]
```diff
@@ -42,26 +42,106 @@ These tools are available from the [R for macOS website](https://mac.r-project.o
...
+Because there is no native way to use the RStudio IDE within WSL2's Ubuntu operating system, you will instead need to install the RStudio Server.
+
+This will provide essentially the same experience as working with RStudio Desktop, except you will access the IDE through your browser instead of through a built-in GUI.
+
+To install RStudio Server, run these commands in the Ubuntu terminal:
+
+1. Download the RStudio Server installer file:
+
+    ```sh
+    wget https://rstudio.org/download/latest/stable/server/jammy/rstudio-server-latest-amd64.deb
```
Comment (jashapiro) [inline docs/technical-setup/environment-setup/install-r-rstudio.md:99]: If we want to be ubuntu-version agnostic
```suggestion
    wget "https://rstudio.org/download/latest/stable/server/$(lsb_release -cs)/rstudio-server-latest-amd64.deb"
```

File: docs/technical-setup/environment-setup/install-r-rstudio.md [hunk 6]
```diff
@@ -42,26 +42,106 @@ These tools are available from the [R for macOS website](https://mac.r-project.o
...
+    sudo apt install --no-install-recommends r-base r-base-dev
+    ```
+
+1. To confirm R was successfully installed, run `R` in the Ubuntu terminal.
+This should launch the R console in terminal; hit `Ctrl+D` to quit.
+    - If you instead get an error message that the command was not found, your R installation was not successful; please feel free to use the [Troubleshooting GitHub Discussions](../../communications-tools/index.md#ask-questions) to get some help!
+
+1. Finally, copy and paste this line into the Ubuntu terminal.
+    ```sh
+    echo "\noptions(repos = list(CRAN='https://p3m.dev/cran/__linux__/jammy/latest'))" >> ~/.Rprofile
```
Comment (jashapiro) [inline docs/technical-setup/environment-setup/install-r-rstudio.md:71]: Echo in my testing on ubuntu leaves the literal `\n`. I don't think it should be needed in most cases, especially with new installs like this.
```suggestion
    echo "options(repos = list(CRAN='https://p3m.dev/cran/__linux__/jammy/latest'))" >> ~/.Rprofile
```

If you really want it, `echo -e` will be safer, or `echo ""; echo "next line"`

Finally, you could make this version agnostic too:

```
 echo "options(repos = list(CRAN='https://p3m.dev/cran/__linux__/$(lsb_release -cs)/latest'))" >> ~/.Rprofile
```

**Review by jashapiro — COMMENTED** [review]
Looks like a good start. I have a few small comments that probably require testing in the WSL environment. They work in previous ubuntu instances, but I can't be sure what the defaults are for wsl

File: docs/technical-setup/environment-setup/install-r-rstudio.md (hunk 1 above)
Comment (sjspielman) [reply docs/technical-setup/environment-setup/install-r-rstudio.md:58]: I had that originally but changed it to jammy at the last minute 😬

File: docs/technical-setup/environment-setup/install-r-rstudio.md (hunk 4 above)
Comment (sjspielman) [reply docs/technical-setup/environment-setup/install-r-rstudio.md:122]: Yeah, probably not.

File: docs/technical-setup/environment-setup/install-r-rstudio.md [hunk 7]
```diff
@@ -42,26 +42,105 @@ These tools are available from the [R for macOS website](https://mac.r-project.o
...
+    sudo apt install --no-install-recommends r-base r-base-dev
+    ```
+
+1. To confirm R was successfully installed, run `R` in the Ubuntu terminal.
+This should launch the R console in terminal; hit `Ctrl+D` to quit.
+    - If you instead get an error message that the command was not found, your R installation was not successful; please feel free to use the [Troubleshooting GitHub Discussions](../../communications-tools/index.md#ask-questions) to get some help!
+
+1. Finally, copy and paste this line into the Ubuntu terminal.
+    ```sh
+    echo -e "options(repos = list(CRAN='https://p3m.dev/cran/__linux__/$(lsb_release -cs)/latest'))" >> ~/.Rprofile
```
Comment (jashapiro) [inline docs/technical-setup/environment-setup/install-r-rstudio.md:71]: Without the `\n` there is nothing for `-e` to do here
```suggestion
    echo "options(repos = list(CRAN='https://p3m.dev/cran/__linux__/$(lsb_release -cs)/latest'))" >> ~/.Rprofile
```

File: docs/technical-setup/environment-setup/install-r-rstudio.md [hunk 8]
```diff
@@ -42,26 +42,105 @@ These tools are available from the [R for macOS website](https://mac.r-project.o
...
+    sudo rstudio-server start
+    ```
+
+- Stop RStudio Server
+
+    ```sh
+    sudo rstudio-server stop
+    ```
+
+To use the RStudio Server, navigate to `localhost:8787` in your browser.
```
Comment (jashapiro) [inline docs/technical-setup/environment-setup/install-r-rstudio.md:135]: I feel like the full url is worth including. 
```suggestion
To use the RStudio Server, navigate to `http://localhost:8787` in your browser.
```

**Review by jashapiro — APPROVED** [review]
LGTM

**Pull request merged**